docs/en_US/NAS/Benchmarks.md

```diff
@@ -7,40 +7,25 @@
     Example Usages <BenchmarksExample>
 ```
 
+## Introduction
+To imporve the reproducibility of NAS algorithms as well as reducing computing resource requirements, researchers proposed a series of NAS benchmarks such as [NAS-Bench-101](https://arxiv.org/abs/1902.09635), [NAS-Bench-201](https://arxiv.org/abs/2001.00326), [NDS](https://arxiv.org/abs/1905.13214), etc. NNI provides a query interface for users to acquire these benchmarks. Within just a few lines of codes, researcher are able to evaluate their NAS algorithms easily and fairly by utilizing these benchmarks.
```

Comment:
NDS and etc.

Reply:
it seems that "and" should not be in front of etc. Refer to https://www.sohu.com/a/149363473_814105.